### DISCUSSION TITLE
Hsteampipe

### DISCUSSION THREAD

**Discussion by promo55** [SteamRE/SteamKit#985] [General]
How can I get the hsteampipe id? Analogue of SteamAPI_GetHSteamPipe () in OSW.

**Comment by m4dEngi** [comment]
In OSW you need to implement this method by yourself. So first you need to initialize steamclient library and create said pipe...

And it's not related to steamkit.

edit: ok i missed the 'analogue' part lol

**Comment by promo55** [comment]
In osw, I have HSteamPipe hSteamPipe = SteamAPI_GetHSteamPipe (). I am wondering if there is a similar implementation in steamkit2

**Comment by psychonic** [comment]
No. SteamKit does not interact with the local Steam client instance.
